tests/docs_src/test_import.py

```diff
@@ -37,8 +37,8 @@ def test_submodules(module: str) -> None:
                 return
             elif (
                 module.startswith("docs_src.user_guide.ui.mesop")
-                or module == "docs_src.tutorial.giphy"
-                or module == "docs_src.tutorial.whatsapp"
+                or module == "docs_src.tutorials.giphy"
+                or module == "docs_src.tutorials.whatsapp"
```

Comment:
Why `tutorials`? It is `tutorial` only - https://github.com/airtai/fastagency/tree/main/docs/docs_src/tutorial. 
The path is literally `docs/docs_src/tutorial/`.

Or am I missing something?

Reply:
Nevermind. I looked at single commit not whole changes. Stupid github.